pkg/api/const.go

```diff
@@ -61,6 +61,8 @@ const (
 	Kubernetes162 OrchestratorVersion = "1.6.2"
 	// Kubernetes166 is the string constant for Kubernetes 1.6.6
 	Kubernetes166 OrchestratorVersion = "1.6.6"
+	// Kubernetes166 is the string constant for Kubernetes 1.6.6
+	Kubernetes170 OrchestratorVersion = "1.7.0"
 	// KubernetesLatest is the string constant for latest Kubernetes version
 	KubernetesLatest OrchestratorVersion = Kubernetes166
```

Comment:
Makes sense. Was just wondering, due to the name of the variable + the variable description in the line above, that points to the fact that it is referring to the latest version and not the "default version" or whatever is a more suited name.